intellij-toml/src/main/kotlin/org/toml/lang/psi/TomlPsiFactory.kt

```diff
@@ -30,6 +30,8 @@ class TomlPsiFactory(private val project: Project, private val markGenerated: Bo
     private inline fun <reified T : PsiElement> PsiElement.descendantOfTypeStrict(): T? =
         PsiTreeUtil.findChildOfType(this, T::class.java, /* strict */ true)
 
+    fun createLiteral(value: String): TomlLiteral = createFromText("dummy = $value") ?: error("")
```

Comment:
What's the difference from `createValue`?

Reply:
Return type. But they indeed make the same things.
Maybe just drop `TomlPsiFactory#createValue`? We use it only in tests and have methods for all other `TomlValue`s: `TomlArray` and `TomlInlineTable`